pkg/binary/envoy/controlplane/istio.go

```diff
@@ -102,6 +103,7 @@ func generateIstioConfig(e *envoy.Runtime) meshconfig.ProxyConfig {
 	cfg.EnvoyAccessLogService = &meshconfig.RemoteService{Address: e.Config.ALSAddresss}
 	// Required: Defaults to MUTUAL_TLS, but we don't configure auth, yet, so it has to be set to NONE
 	cfg.ControlPlaneAuthPolicy = meshconfig.AuthenticationPolicy_NONE
+	cfg.Tracing = nil // Prevent server from hanging on unavailable Zipkin cluster
```

Comment:
there were two reasons why the istio test hung, one was some bug in envoy with XDS, the other was this. Envoy wouldn't become available because there was no zipkin host ready.